### PR TITLE
Update to 1.0 with Sylius 1.8 and Rich Editor 2.0

### PR DESCRIPTION
**Pull request by maximehuran**

### CODE REVIEW

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: src/Entity/Page.php [hunk 1]
```diff
@@ -29,15 +41,15 @@ class Page implements PageInterface
...
+     * @ORM\Id
+     * @ORM\GeneratedValue
      * @ORM\Column(type="integer")
      */
     private $id;
 
     /**
      * @var bool
-     * @ORM\Column(type="boolean", options={"default": true})
+     * @ORM\Column(type="boolean", options={"default"=true})
```
[the remark this answers is not in the document]
Comment (maximehuran) [reply src/Entity/Page.php:52]: PHP CS I think

File: src/Entity/Page.php [hunk 2]
```diff
@@ -48,10 +60,10 @@ class Page implements PageInterface
     protected $code;
 
     /**
-     * @var Collection
+     * @var Collection<int, ChannelInterface>
      * @ORM\ManyToMany(targetEntity="\Sylius\Component\Channel\Model\Channel")
      * @ORM\JoinTable(
-     *     name="mbiz_cms_page_channels",
+     *     name="monsieurbiz_cms_page_channels",
```
[the remark this answers is not in the document]
Comment (maximehuran) [reply src/Entity/Page.php:66]: Why ? I prefer to have `monsieurbiz` on the table name instead of `mbiz`

[the remark this answers is not in the document]
Comment (jacquesbh) [reply src/Entity/Page.php:66]: Ok, then an `UPGRADE-1.0.md` is welcomed!

[the remark this answers is not in the document]
Comment (maximehuran) [reply src/Entity/Page.php:66]: Done

**Review by jacquesbh — APPROVED** [review]

**Comment by jacquesbh** [comment]
Big one!

Thank you Maxime 🎉.

We'll still have to tag when the Rich Editor v2 is out!

**Pull request merged**